roles/sideload_kernel/files/flip_kernel

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+NONRT=(kernel{-core,-modules,-modules-core,-modules-extra})
+RT=(kernel-rt{-core,-modules,-modules-core,-modules-extra})
+PREFETCH=/tmp
+
+replace_standard_kernel() {
+    local -a install
+    for pkg in "$@"; do
+        install+=(--install "${pkg}")
+    done
+    sudo rpm-ostree override remove kernel "${NONRT[@]}" "${install[@]}"
```

Comment:
You can get rid of the assorted `sudo` calls in this script if you force it to run as root e.g. adding `if $( id -u ) -neq 0; then exit 1; fi` or similar at the very top of the script. I believe this script should be run as root anyway.

Reply:
The script was originally designed to run by hand, and envolved out of something that was information as non-root and effective as root.  But it's true that since the meat of the script needs root, the whole script should. I'll do that.